mc/Diameter.package/AVPCollection.class/instance/printOn..st

```diff
@@ -0,0 +1,4 @@
+initialization
+printOn: aStream
+	super printOn: aStream.
+	elements printElementsOn: aStream.
```

Comment:
Is this unrelated? How do we know the elements belonged to the AVPCollection?

Reply:
I'm not sure if I get you right, but `AVPCollection` is instantiated only by `AVPPath` which is based on the `roots` AVPs of a diameterMessage. Does that answer you question? Feel free to elaborate or rephrase if I didn't answer what you wanted to know

Reply:
When I saw collection I would probably want to see output like "AVPCollection< Elem1, Elem2, Elem3>"?  (< being just any marker). It's not critical.

Reply:
Ah I see what you mean. Is already doing it (the `super printElementsOn:` had that in mind 🙂 ).
They currently look like this when you `printIt`:
`an AVPCollection(AVPHeader 282 AVPHeader 282 AVPHeader 282 AVPHeader 282)`

Good?

Reply:
Good enough. Commas might be nice. :)